test/compile/layout.test.ts

```diff
@@ -44,7 +44,7 @@ describe('compile/layout', () => {
       });
 
       const sizeExpr = unitSizeExpr(model, X);
-      assert.equal(sizeExpr, 'max(datum["distinct_a"] + 2*0.5, 0) * 21');
+      assert.equal(sizeExpr, 'max(datum["distinct_a"] - 1 + 2*0.5, 0) * 21');
```

Comment:
I'm NOT gonna add comments to unit tests unless it's necessary.  
This is a fairly small method, you can easily read the code which is already well commented. 

Plus, I'm not changing the 0.5 part in this PR.